api/api_common.go

```diff
@@ -46,6 +47,11 @@ type Common interface {
 	// usage and current rate per protocol
 	NetBandwidthStatsByProtocol(ctx context.Context) (map[protocol.ID]metrics.Stats, error)
 
+	// ConnectionGater API
+	NetBlockAdd(ctx context.Context, acl dtypes.NetBlockList) error
+	NetBlockRemove(ctx context.Context, acl dtypes.NetBlockList) error
+	NetBlockList(ctx context.Context) (dtypes.NetBlockList, error)
```

Comment:
I would prefer that API did not depend on `dtypes`.

Reply:
Sure, but where should I put the ACL type definition then?

Reply:
Moved the struct to `api/types.go` and removed the dependency.